src/libraries/System.Private.CoreLib/src/System/ArgumentException.cs

```diff
@@ -1,25 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-/*=============================================================================
-**
-**
-**
-** Purpose: Exception class for invalid arguments to a method.
-**
-**
-=============================================================================*/
-
 using System.ComponentModel;
 using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
 using System.Runtime.Serialization;
 
 namespace System
 {
-    // The ArgumentException is thrown when an argument does not meet
-    // the contract of the method.  Ideally it should give a meaningful error
-    // message describing what was wrong and which parameter is incorrect.
+    /// <summary>
+    /// The exception that is thrown when a null reference (Nothing in Visual Basic) is passed to a method that does not accept it as a valid argument.
+    /// </summary>
```

Comment:
This comment would +- apply to ArgumentNullException, not to the more generic ArgumentException.